docs/docson/build-schema.json

```diff
@@ -350,6 +350,19 @@
                 }
             }
         },
+        "react-specs": {
+            "type": "object",
+            "properties": {
+                "jsx": {
+                    "$ref": "#/definitions/jsx-version",
```

Comment:
I think this should just mention a number, not rescript-react.
It's the JSX version number, independently from the fact V3 can be used almost only from rescript-react.

Reply:
Fixed https://github.com/rescript-lang/rescript-compiler/pull/5484/commits/81029b30f20e8b12b9dff1229228e31daa5e7c5b